src/components/Header/Header.js

```diff
@@ -2,14 +2,32 @@ import React from 'react';
 import logo from '../../bridgeLogo.svg';
 import { HeaderStyle } from './HeaderStyle';
 import { LogoStyle } from './LogoStyle';
-import { Wrapper } from '../../GlobalStyle';
+import { Wrapper, Badge } from '../../GlobalStyle';
+import { useHistory } from 'react-router-dom';
 
 export const Header = () => {
+  const history = useHistory();
   return (
     <HeaderStyle>
-      <Wrapper bold white fontMedium flex align="center" row>
-        <LogoStyle src={logo} alt="Logo" />
-        Cohort Application Data
+      <Wrapper flex="space-between">
+        <Wrapper bold white fontMedium flex align="center" row>
+          <LogoStyle
+            src={logo}
+            alt="Logo"
+            onClick={() => history.push('/home')}
+            pointer
+          />
```

Comment:
shouldn't this be wrapped in a `Link` instead of `onClick`?

Reply:
I can change it to something like this: https://www.w3.org/TR/wai-aria-practices-1.1/examples/link/link.html
you think it will be better?

Reply:
Agreed - I think instead of pushing to the history object if it's wrapped in a react-router Link, the Link will be rendered as an HTML anchor tag.